<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DB_AccountManagement</class>
 <widget class="QWidget" name="DB_AccountManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>609</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="DB_CustomFrame_3" name="frame_loading">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_loading {
border: none;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="DB_CustomMessageBox" name="messageBox" native="true">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>441</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(78, 155, 252);
border-top-left-radius: 8px;
border-top-right-radius: 8px;</string>
   </property>
   <property name="text">
    <string>账号管理</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="DB_CustomLabel_4" name="label_userIcon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>56</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>83</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 85, 85);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_telephone">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 85, 85);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DB_CustomLabel_1" name="label_cancellation">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(185, 185, 185);
color: rgb(165, 165, 165);
border-radius: 3px;</string>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_weixinLogo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>44</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(&quot;:/images/courseware/weixin-2@2x.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DB_CustomLabel_1" name="label_unbundling">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>165</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 155, 252);</string>
   </property>
   <property name="text">
    <string>解绑</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="DB_CustomLabel_1" name="label_closed">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(78, 155, 252);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>461</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 235, 235);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_weixinName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>165</y>
     <width>221</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DB_CustomLabel_1</class>
   <extends>QLabel</extends>
   <header location="global">CustomWidget/db_customlabel.h</header>
  </customwidget>
  <customwidget>
   <class>DB_CustomFrame_3</class>
   <extends>QFrame</extends>
   <header location="global">CustomWidget/db_customframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DB_CustomLabel_4</class>
   <extends>QLabel</extends>
   <header location="global">CustomWidget/db_customlabel.h</header>
  </customwidget>
  <customwidget>
   <class>DB_CustomMessageBox</class>
   <extends>QWidget</extends>
   <header location="global">CustomWidget/db_custommessagebox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
